applications/common/backend/src/main/resources/libs/kyanite/common/components/page/dialog/.content.json

```diff
@@ -55,6 +55,85 @@
         "name": "description",
         "label": "Description"
       }
+    },
+    "metadataTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "Additional Metadata",
```

Comment:
Maybe we could rename it to "Social media metadata"?

Reply:
Done